src/gui/macutilities.mm

```diff
@@ -45,7 +46,7 @@ QPixmap pixmapForExtension(const QString &ext, const QSize &size)
     {
         @autoreleasepool
         {
-            NSImage *image = [[NSWorkspace sharedWorkspace] iconForFileType:ext.toNSString()];
+            NSImage *image = [[NSWorkspace sharedWorkspace] iconForContentType:[UTType typeWithFilenameExtension:ext.toNSString()]];
```

Comment:
Please add `const` if it can compile:
```suggestion
            const NSImage *image = [[NSWorkspace sharedWorkspace] iconForContentType:[UTType typeWithFilenameExtension:ext.toNSString()]];
```